research/delf/delf/python/training/README.md

```diff
@@ -175,6 +175,39 @@ python3 train.py \
   --delg_global_features
 ```
 
+### Training with the Autoencoder
+
+In order to train with the autoencoder described in the DELG paper you can add
+the `--use_autoencoder` parameter for both training with local and local and
+global features:
+
+* Training with local features and the autoencoder:
+
+```
+python3 train.py \
+  --train_file_pattern=gldv2_dataset/tfrecord/train* \
+  --validation_file_pattern=gldv2_dataset/tfrecord/validation* \
+  --imagenet_checkpoint=resnet50_weights_tf_dim_ordering_tf_kernels_notop.h5 \
+  --dataset_version=gld_v2_clean \
+  --logdir=gldv2_training/ \
+  --use_autoencoder
+````
+* Training with local and global features and the autoencoder:
+
+```
+python3 train.py \
+  --train_file_pattern=gldv2_dataset/tfrecord/train* \
+  --validation_file_pattern=gldv2_dataset/tfrecord/validation* \
+  --imagenet_checkpoint=resnet50_weights_tf_dim_ordering_tf_kernels_notop.h5 \
+  --dataset_version=gld_v2_clean \
+  --logdir=gldv2_training/ \
+  --delg_global_features \
+  --use_autoencoder
+```
+
+*NOTE: The `--use_autoencoder` parameter is set by default to `True`, therefore
```

Comment:
actually, it seems like these two commands will lead to the exact same behavior as the ones in the sections `Training with Local Features` and `Training with Local and Global Features`? (since `use_autoencoder` is True by default).

So maybe what I would suggest is to remove this new section and simply mention in the previous two ones to note that use_autoencoder is True by default, so the model is using it.

Reply:
Makes sense. I removed this section and mentioned in each of the other 2 sections that --use_autoencoder is True by default.